rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/DefenceResponseJsonMapper.java

```diff
@@ -57,19 +57,23 @@ private BigDecimal getAmountAdmitted(Response response) {
     }
 
     private JsonObject mapPayment(Response response) {
-        JsonObjectBuilder jsonObjectBuilder = new NullAwareJsonObjectBuilder();
         if (ResponseUtils.isAdmissionResponse(response)) {
-            updateJsonBuilderForAdmissions(jsonObjectBuilder, response);
-            return jsonObjectBuilder.build();
+            JsonObjectBuilder builder = updateJsonBuilderForAdmissions(response);
+            if (builder != null) {
+                return builder.build();
+            }
         }
-
         return null;
     }
 
-    private void updateJsonBuilderForAdmissions(JsonObjectBuilder jsonObjectBuilder, Response response) {
+    private JsonObjectBuilder updateJsonBuilderForAdmissions(Response response) {
```

Comment:
As it's no longer accepting a JsonOB, the `update` prefix isn't really appropriate.